### PR TITLE
fix(electron): downgrade to 25.8.4 COMPASS-7291

### PR DESCRIPTION
**Pull request by kmruiz**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [x] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [x] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by mcasimir — APPROVED** [review]

**Comment by lerouxb** [comment]
I don't know _what_ happened to this package-lock.json file, but you should probably start over :) I'd recommend you checkout the one from main and then do npm install again. Although npm tends to be weird about downgrading things in package-lock.json files..

File: package.json [hunk 1]
```diff
@@ -80,5 +80,8 @@
     "packages/*",
     "configs/*",
     "scripts"
-  ]
+  ],
+  "dependencies": {
+    "electron": "^25.9.0"
+  }
```
Comment (addaleax) [inline package.json:86]: You’ll probably want to undo this change and then run `npm install` again, it should keep the package-lock basically as it was before (except for metadata fields)

Comment (kmruiz) [reply package.json]: Done :D

**Review by addaleax — APPROVED** [review]
